autoPyTorch/api/base_task.py

```diff
@@ -840,10 +934,10 @@ def _search(
             precision (int: default=32):
                 Numeric precision used when loading ensemble data.
                 Can be either '16', '32' or '64'.
-            disable_file_output (Union[bool, List]):
-                If True, disable model and prediction output.
-                Can also be used as a list to pass more fine-grained
-                information on what to save. Allowed elements in the list are:
+            disable_file_output (List[Union[str, DisableFileOutputParameters]]):
```

Comment:
This differs from the actual type signature

Reply:
yeah I fixed them